packages/abi-coder/src/coders/b512.test.ts

```diff
@@ -0,0 +1,97 @@
+import B256Coder from './b512';
```

Comment:
```suggestion
import B512Coder from './b512';
```

Reply:
One of the (many) reasons I dislike default exports is that we can't enforce proper nomenclature to consumers or across our codebase, opening the doors to spelling errors and potentially different variable names on each import.